src/GoogleAnalytics.ts

```diff
@@ -0,0 +1,23 @@
+export const trackEvent = (
+	timingCategory: string,
+	timingVar: string,
+	timingLabel: string,
+	trackerName: string,
+): void => {
+	const { ga } = window;
+
+	if (typeof ga === 'undefined') {
+		console.error("Can't track GA event - GA library not loaded");
+		return;
+	}
+	const timeSincePageLoad: number = Math.round(window.performance.now());
```

Comment:
Hi just wondering why the `window.performance.now()` value is rounded?
From performance api docs it looks like browsers already round to x ms to mitigate timing attacks

Reply:
Hi, we round on frontend as well and wanted to keep it for consistency: https://github.com/guardian/frontend/blob/6f5897587dc2f5b03596e0843dcc229763a32ef2/static/src/javascripts/projects/common/modules/analytics/google.js#L117

But since you pointed out we may not need it? Although be reading the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) it says:
```
To offer protection against timing attacks and fingerprinting, the precision of performance.now() **might** get rounded **depending on browser settings**.
```
I concerned this might not be a constant behaviour?

Reply:
Hi the "might" is concerning!

The W3C spec says browsers _should_ add randomness and also caniuse:

https://www.w3.org/TR/2019/REC-hr-time-2-20191121/#sec-domhighrestimestamp
https://caniuse.com/high-resolution-time

But you're right, best to be cautious 👍